fix_includes.py

```diff
@@ -2169,6 +2172,16 @@ def main(argv):
   parser.add_option('--nosafe_headers', action='store_false',
                     dest='safe_headers')
 
+  # --reorder and --no-reorder share the same destination variable.
+  parser.add_option('--reorder', action='store_true', dest='reorder',
+                    help=('Re-order lines relative to other similar lines '
+                          '(eg, headers relative to other headers)'))
+
+  # --reorder and --no-reorder share the same destination variable.  Omitting
+  # the default here would make the default True.
+  parser.add_option('--no-reorder', default=False, action='store_false', dest='reorder',
+                    help=('Do not re-order lines relative to other similar lines.'))
```

Comment:
No need for parentheses around the `help` string, parenthesized strings are only useful for multiline strings.